### ISSUE TITLE
Windows CI

### ISSUE THREAD

**Issue by harpocrates** [haskell/haddock#1149]
It would be nice to have some CI for windows. We've historically had some serious windows-only issues that have crept in. The two obvious options I see:

   * Github actions: this would be pretty easy to set up, but I want to check it is OK since I think it would drain minutes from the `haskell` github organization account (this set of minutes is shared among all org projects, I think)
 
   * AppVeyor: I did the work to set this in https://github.com/haskell/haddock/pull/1096 up a while ago (on my personal AppVeyor account - similar to what Cabal was doing at the time for windows CI), but I don't have the github permissions to register the webhook on the Haddock repo

/cc @cartazio @emilypi @hvr

**Comment by harpocrates** [comment]
^ Ping

**Comment by emilypi** [comment]
I agree - i think it'd be useful! Unfortunately I've never set up Windows CI before, so i'm useless. Punting to @hvr or @cartazio
